HotelManagerment/com.spring.sample/src/com/spring/sample/entity/RoomType.java

```diff
@@ -3,15 +3,32 @@
 import java.io.Serializable;
 import java.util.Set;
 
-@SuppressWarnings("serial")
-public class RoomType extends BaseEntity implements Serializable{
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+
+import org.hibernate.annotations.GenericGenerator;
+
+@Entity(name = "ROOM_TYPES")
+public class RoomType extends BaseEntity implements Serializable {
+	@Id
+	@Column(name = "RoomTypeID")
+	@GenericGenerator(name = "assigned-generator", strategy = "assigned")
+	@GeneratedValue(generator = "assigned-generator")
 	private String roomTypeID;
```

Comment:
```suggestion
	private String id;
```
tự check lại các field bên dưới